### DISCUSSION TITLE
Stock firmware needed for Yi Outdoor Camera

### DISCUSSION THREAD

**Discussion by J1GG4** [alienatedsec/yi-hack-v5#359] [Q&A] [answered]
Hello!

I want to reflash the stock firmware on my Yi Outdoor Camera, but I can't find it anywhere.
It is not available to download on the official site: https://www2.yitechnology.com/firmware/index

There is a README file here in the stock_firmware folder containing:

> _stock_firmware
> 
> Why an empty folder? Because I'm not sure if I can re-distribute on GitHub the original Xiaomi firmware images needed to create yi-hack-v4.
> 
> You can download the images from mega.nz (link coming soon)_

Can you provide me the stock firmware, please? Much appreciated. :)

![image](https://github.com/alienatedsec/yi-hack-v5/assets/27810325/ef334e6a-0f71-443c-a25a-4655c4b7a46d)

**Comment by g1za** [comment, marked as answer]
Have a look [here](https://github.com/shadow-1/yi-hack-v3?tab=readme-ov-file#going-back-to-stock-firmware) and see if you find what you need.
`yi-hack-v3 -> Recovery`

**Reply by J1GG4** [reply]
Thanks a bunch! Got it, you're awesome. :)
